spec/factories/http_mappings.rb

```diff
@@ -1,6 +1,6 @@
 FactoryGirl.define do
   factory :http_mapping, class: 'Atmosphere::HttpMapping' do |f|
-    url  Faker::Internet.domain_name
+    url  FFaker::Internet.domain_name
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.